build/run-e2e-tests-policy-framework-prow.sh

```diff
@@ -27,6 +27,10 @@ echo "* Install cert manager"
 $DIR/install-cert-manager.sh
 
 echo "* Set up cluster for test"
+export VERSION_TAG="latest"
+if [ "${PULL_BASE_REF}" ] && [ "${PULL_BASE_REF}" != "main" ]; then
+  export VERSION_TAG="${VERSION_TAG}-${PULL_BASE_REF#*-}"
```

Comment:
Can you add a comment (or something) with an example of what the result will be from this? I think it's `latest-2.4`, but I'm not actually positive.